sky/templates/kubernetes-port-forward-proxy-command.sh.j2

```diff
@@ -45,8 +45,26 @@ done
 # To avoid errors when many concurrent requests are sent (see https://github.com/skypilot-org/skypilot/issues/2628),
 # we add a random delay before establishing the socat connection.
 # Empirically, this needs to be at least 1 second. We set this to be random between 1 and 2 seconds.
-sleep $(shuf -i 10-20 -n 1 | awk '{printf "%f", $1/10}')
-
+if which shuf > /dev/null; then
+  sleep $(shuf -i 10-20 -n 1 | awk '{printf "%f", $1/10}')
```

Comment:
We need to add shuf/gshuf checks to `kubernetes_utils.py::check_port_forward_mode_dependencies()` as well, and include installation commands there.